mmv1/third_party/terraform/utils/node_config.go.erb

```diff
@@ -638,6 +647,7 @@ func flattenNodeConfig(c *container.NodeConfig) []map[string]interface{} {
 		"labels":                   c.Labels,
 		"tags":                     c.Tags,
 		"preemptible":              c.Preemptible,
+		"spot":                     c.Spot,
```

Comment:
Linting error:

```
c.Spot undefined (type *container.NodeConfig has no field or method Spot)
```

Reply:
Thank you, this is because of `container/v1beta1` only available in beta, and ga uses `container/v1`:
https://github.com/GoogleCloudPlatform/magic-modules/blob/master/mmv1/third_party/terraform/utils/node_config.go.erb#L11
So, I've guarded all `spot` calls to be only available in beta.